### ISSUE TITLE
[Project] Aircraft-Wildlife Collisions

### ISSUE THREAD

**Issue by angelareplica** [jsoma/data-studio-projects#235]
Please complete all of the following sections, or the ghost of Joseph Pulitzer will spookily dance around your issue! A completed version of this template can be found at https://github.com/jsoma/data-studio-projects/issues/1

# Pitch

## Summary

The U.S. Federal Aviation Association has a database of wildlife strikes from 1990 through April 2018. I'd like to explore the database in hopes of answering several questions:
-What species are involved in the most accidents/collisions?
-How much have collisions cost? What species are the most expensive to hit?
-When/where do most collisions occur?
-There's a general upward trend in collisions -- why?

## Details

**Possible headline(s):** 
Colliding With Wildlife Has Cost Airlines 588 Million Dollars Since 1990

**Data set(s):** 
https://wildlife.faa.gov/databaseSearch.aspx

**Code repository:** 
https://github.com/angelareplica/data-studio/tree/master/code/04-aircraft-wildlife-strikes
	
**Possible problems/fears/questions:** 
-There's A LOT to dig into in the database, and my biggest struggle is trying to decide on what to focus on. 
-I'd like to overlay a line graph of total flights per year with my bar chart of total strikes per year, but I haven't been able to find reliable data source for that. It's hard to find a correlation or explanation for the upward trend in collisions without that data. 
-I'd also love to be able to categorize species into birds/terrestrial animals/etc, but unfortunately I don't think this is feasible.

## Work so far
I've done some cursory analysis and made some ugly/basic charts.

![aircraft-wildlife](https://user-images.githubusercontent.com/20685568/43435418-dd18005c-944d-11e8-9fbe-6fc10da288dd.png)

![aircraft-wildlife-airlines](https://user-images.githubusercontent.com/20685568/43435426-e51b11e0-944d-11e8-9abd-2adcc9abd0f2.png)

![aircraft-wildlife-airports](https://user-images.githubusercontent.com/20685568/43435439-ec403ac2-944d-11e8-97f3-a16ead26aa3b.png)

![aircraft-wildlife-species](https://user-images.githubusercontent.com/20685568/43435443-f056ab50-944d-11e8-88f8-7c098f42f5fa.png)

## Checklist

This checklist must be completed before you submit your draft.

- [x] I have already spent time with my data set, opening it, exploring it, etc
- [x] I have created a "DIARY.md" file to save links and list all of the terrible, no good problems I come across
- [x] My issue links to my data set(s)
- [x] My issue links to my code repository
- [x] My issue explains what I'd like to explore in the data set
- [x] My issue includes images - either inspiration or what I've done so far

**Comment by cfelke** [comment]
I agree with you that it would be helpful to see the ratio of flights and aircraft-wildlife collisions to get a better understanding of the upward trend. I have two minor remarks: The graphs would be much easier to read if you showed them in descending order and the colours which keep repeating themselves are a bit confusing. When comparing the count of collisions with the different airlines, I'd swap the axis so you can 1) easily read the name of each airline on the left and 2) show the numbers at the top.

I'm curious about the visualization you're gonna end up with.

**Comment by angelareplica** [comment]
Update 1

# Update
	
## Your project content: 
I've decided to focus primarily on cost of repairs.

Money spent on aircraft repairs by year:
![repairs_cost_edited](https://user-images.githubusercontent.com/20685568/43622336-9ed55d5e-96a9-11e8-913e-d8783a8dba41.png)

Collisions that required aircraft repairs most often hit these animals:
![species_cost_edited](https://user-images.githubusercontent.com/20685568/43622349-b0b63fb6-96a9-11e8-9175-c00fb81929b5.png)

## Any changes in direction or topic?
Nope.
  
## Problems/Questions
Multiple columns in the data set don't match the documentation. There are several things I'd like to chart but can't find a feasible way to. I'd like to create stacked bar charts showing what types of animals get hit (birds/terrestrial animals/reptiles), and a stacked bar chart showing what kinds of aircraft are involved in wildlife strike (commercial/military/private).
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [x] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by dz2383** [comment]
Love the color scheme!
Here are my suggestions:
-would it be better to show some number the highest and lowest number on the chart?
-since the second graph is "barh," maybe change the first one to "barh" as well, which will make it  easier to read years
-For the second graph, what's the unit for 200, 400, 600?

**Comment by sarahslo** [comment]
love that you are drawn to these quirky datasets.

small thing there, the annotations should point toward the data, not vice-versa. the 'usually' note should be close to the data not forced into the space by the arrow.

i wonder about the 'unknown bird' category. it's kind of funny there is a small, medium and large and the medium shows up the most. what is the 'unknown bird' data without a size? maybe anything unknown is a different color bar? it's an almost comical part of the data set because all they likely get is bird parts.

**Comment by angelareplica** [comment]
Thanks for the feedback above!
Update #1:

# Update
	
## Your project content: images/words/etc
	
Money spent on aircraft repairs by year:
![repairs_cost_edited](https://user-images.githubusercontent.com/20685568/43814445-72965de0-9a98-11e8-8c9a-432f6bca8171.png)

Collisions that required aircraft repairs most often hit these animals:
![species_cost_edited_2](https://user-images.githubusercontent.com/20685568/43814452-7c4c2f9a-9a98-11e8-885c-b36b8263ca31.png)

Types of animals hit every year, broken down by whether they fly or not: 
![species_distribution_edited](https://user-images.githubusercontent.com/20685568/43814459-801b81b6-9a98-11e8-9ec9-c630d5aee3b0.png)

  
## Any changes in direction or topic?
Nope.
	
## Problems/Questions
 I looped through every species listed to assign "terrestrial" or "bird/bat" to it. I think I got mostly everything, but it's definitely possible that I missed some terrestrial birds or overlooked something, since there were hundreds of species. I plan to do the same thing to categorize types of aircraft (commercial, military, private, etc).
	
## Checklist
	
- [x] I have included my visuals
- [x] I have filled out the sections above
- [ ] I have been updating my DIARY.md with details about my process
- [ ] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by angelareplica** [comment]
# Final

## Project visuals/text

Money spent on aircraft repairs per year
![repairs_cost_edited](https://user-images.githubusercontent.com/20685568/43986836-b4147474-9ce5-11e8-9957-2f30178d4ee5.png)

Aerial or Terrestrial? The kinds of animals struck by aircraft every year
![species_distribution_edited_2](https://user-images.githubusercontent.com/20685568/43986839-d0ad92b4-9ce5-11e8-9fbd-c8ac66bab139.png)

The kinds of animals struck by aircraft every year resulting in repair costs
![species_distribution_repairs_edited_2](https://user-images.githubusercontent.com/20685568/43986844-dffbdd5c-9ce5-11e8-905a-9c74e6424936.png)

Species that are expensive to hit
![species_cost_edited_2](https://user-images.githubusercontent.com/20685568/43986862-48431d62-9ce6-11e8-8c14-3a14d9e7a4df.png)


## Details

**Headline:**  
Colliding With Wildlife Has Cost Airlines Over 588 Million Dollars Since 1990

**Published website version:**
https://angelareplica.github.io/ds-wildlife-strikes/

**Code repository:**
https://github.com/angelareplica/data-studio/tree/master/code/04-aircraft-wildlife-strikes

**Final data set(s):** 
https://github.com/angelareplica/data-studio/tree/master/code/04-aircraft-wildlife-strikes

## What did you find to be the most difficult part of this project?
Deciding what to focus on, and how in-depth to go. There's SO much information in the dataset.

## Are you satisfied with what you produced? Is there anything you would like to change or improve?
I like my charts, but I feel like there's much more I could have done with the dataset! Hope to do more work with it. I also only stuck with bar charts on this one, so I tried to make them cooler to look at -- but perhaps I could have done more.

## Checklist

- [x] I have included my visuals
- [x] I have posted my project to my project website
- [x] I have been updating my DIARY.md with details about my process
- [x] I have uploaded/updated any Jupyter Notebooks or other datasets into my code repository

**Comment by christina10211** [comment]
Hi! The bot has brought me here. I really love your color scheme and the patterns you applied to the color, it's lovely! I totally share the same feeling when you have a huge dataset and get lost in the information, finding an angle becomes soooo difficult. I at first got confused between the second and third charts, but then I realized the difference. Maybe when you revisit this project, you can try to apply another data representation or bring in more angles. Good job!!!
